platform/mds/mds-test-bundle/src/test/java/org/motechproject/mds/test/osgi/MdsDdeBundleIT.java

```diff
@@ -1757,6 +1758,7 @@ public void shouldCreateHistoryForOneToOneRelationships() {
         house.setName("A house");
         house.setHouseNumber((short)12);
         house.setAddress(address);
+        house.setUId(UUID.randomUUID());
```

Comment:
can you add a assert to check if the uuid field has a correct value ? You can save randomUUID in separately object